docs/src/appendix/snfoundry-toml.md

```diff
@@ -0,0 +1,101 @@
+# The Manifest Format
+
+The `snfoundry.toml` file is not included in every package by default. If you want to use it, you need to create it yourself. 
+This file allows you to specify sncast settings. It is written in the [TOML](https://toml.io/) format. 
+It contains metadata needed to configure sncast settings and should be placed in the root of your project. 
```

Comment:
In fact it's searched for in the project root and above in the filesystem. Maybe add another line explaining the `sncast`'s behaviour in case of lack of the file?